### PR TITLE
v3.10 changelog

### PR DESCRIPTION
**Pull request by JeffSchering**

### CODE REVIEW

**Comment by esethna** [comment]
Done except for known issues which will be updated after release testing

**Comment by esethna** [comment]
Updated known issues

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+#### New and Improved Keyboard Shortcuts
+- Redesigned channel switcher (CTRL/CMD+K)
+- Browse direct and group message channels (CTRL/CMD+SHIFT+K) and reply to the most recent message (SHIFT+UP) with new shortcuts.
+
+### Improvements
+
+#### Web User Interface
+- Enter key now confirms deletion on the screens to delete a custom emoji and delete a channel
+- Team and channel URLs now replace accented characters with their ASCII equivalents
+- Recent menitons and flagged posts icons in the header are now highlighted when they are active in the right-hand sidebar
```
Comment (jasonblais) [inline source/administration/changelog.md]: menitons --> mentions

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Enter key now confirms leaving a team from the Leave Team modal
+- Profile popover now opens when clicking a username in mobile browser view
+- /join now allows switching to a private channel to which the user has access
+- Improved the formatting of Mattermost content when copying and pasting to other apps
+- Added the ability for users to view and modify their online status from their profile picture in the header
+
+#### Notifications
+- Added "@here" to the list of channel-wide mentions in Account Settings
+- Added a reminder when your Mattermost window is refreshed if a status override slash command in used to set yourself as /away or /offline 
+- Users will see a confimation dialog when attempting to use @all or @channel in a channel with over 10 users
```
Comment (jasonblais) [inline source/administration/changelog.md]: actually it's "over 5 users" matching Slack

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Channel Admins can now delete users messages within their channel if permitted in the System Console.
+- Errors are now logged when failing to load config through the command line
+- Reduced unnecessary database reads and writes when bulk importing users
+
+#### System Console
+- System Console main dropdown menu now has links to the Admin Guide, Troubleshooting Forum, Commercial Support Page and the About Mattermost dialog
+- Added the ability to enable Legacy Signature (AWS Signature V2) with S3 compatible servers
+
+#### Authentication
+- Added a redirect to the appropriate team or channel if navigating to a Mattermost URL when logged out
```
Comment (jasonblais) [inline source/administration/changelog.md]: it actually works for posts too (e.g. if you click to a permalink, then log in, it'll redirect you to the correct post).

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Team Invite links now open in the current tab
```
Comment (jasonblais) [inline source/administration/changelog.md]: Invite --> invite

Also, I think the user joins the team in all active sessions, not just the one that gets clicked

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Performance
+- Upgraded GORP to support connection timeouts on MySQL and missing database columns on MySQL and Postgres
+
+#### Integrations
+- Posts from webhooks that are greater than 4000 characters are now broken into multiple posts
+
+#### Enterprise Edition
+- Added an announcement banner for Admins to make maintenance announcements across the entire system configurable in the System Console
```
Comment (jasonblais) [inline source/administration/changelog.md]: minor: It sounds like the banner is for admins only, although it's for end users, which the admin can configure

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+### Bug Fixes
+- Dragging and dropping a file onto the left-hand sidebar no longer navigates away from Mattermost to open the file in the browser
+- Textbox will no longer overlap the center pane message area as it expands when typing
+- Fixed an issue where statuses could get stuck online after quitting the desktop app or closing the browser window
```
Comment (jasonblais) [inline source/administration/changelog.md]: was there any follow-up on this post: https://pre-release.mattermost.com/core/pl/ospjkocq9jf6dn4uty4xpat7oc

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Fixed the positioning of the pin icon and channel header on Edge
+
+### Compatibility  
+
+#### Removed and deprecated features
+- System Console settings in **Files > Images** scheduled for removal in July 2017 release. This includes:
+  - Image preview height and width
+  - Profile picture height and width
+  - Image thumbnail height and width
+- Account Settings options for **Display** > ** Display Font** and **Display** > **Teammate Name Display** are scheduled for removal in July 2017 release.
```
Comment (jasonblais) [inline source/administration/changelog.md]: formatting typo

File: source/administration/changelog.md [hunk 8]
```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+Many thanks to all our contributors. In alphabetical order:
+
+/platform
+
+- [asaadmahmood](https://github.com/asaadmahmood), [coreyhulen](https://github.com/coreyhulen), [cpanato](https://github.com/cpanato), [crspeller](https://github.com/crspeller), [dmeza](https://github.com/dmeza), [doh5](https://github.com/doh5), [enahum](https://github.com/enahum), [grundleborg](https://github.com/grundleborg), [harshavardhana](https://github.com/harshavardhana), [hmhealey](https://github.com/hmhealey), [jasonblais](https://github.com/jasonblais), [jwilander](https://github.com/jwilander), [kulak-at](https://github.com/kulak-at), [saturninoabril](https://github.com/saturninoabril), [tjuerge](https://github.com/tjuerge)
+
+/docs
+
+- [cpanato](https://github.com/cpanato), [crspeller](https://github.com/crspeller), [esethna](https://github.com/esethna), [hmhealey](https://github.com/hmhealey),  [it33](https://github.com/it33), [jasonblais](https://github.com/jasonblais), [JeffSchering](https://github.com/JeffSchering), [jwilander](https://github.com/jwilander), [kjkeane](https://github.com/kjkeane), [lindy65](https://github.com/lindy65)
```
Comment (jasonblais) [inline source/administration/changelog.md]: MikeDaniel18 as well https://github.com/mattermost/docs/pull/1200

File: source/administration/changelog.md (hunk 6 above)
Comment (esethna) [inline source/administration/changelog.md]: Hmm, I will try to repro some more

**Comment by esethna** [comment]
+1, Verified by JW the one is included already

**Comment by esethna** [comment]
@JeffSchering can you please help do an editor review pass and then it is ready to merge

**Pull request merged**